testing/tests/DevExpress.ui.widgets.scheduler/common.tests.js

```diff
@@ -330,77 +319,131 @@ QUnit.testStart(function() {
+                startDate: new Date(2015, 4, 24, 9),
+                endDate: new Date(2015, 4, 24, 11),
+                text: "Task 1"
             }],
             height: 600,
             currentDate: new Date(2015, 4, 24),
-            displayedAppointmentDataField: "Field",
             currentView: "month",
             views: ["month"],
-            appointmentTooltipTemplate: function(data) {
-                assert.deepEqual(data.Field, {
-                    startDate: new Date(2015, 4, 25, 9, 10),
-                    endDate: new Date(2015, 4, 25, 11, 1),
-                    allDay: true,
-                    recurrenceRule: "FREQ=DAILY;COUNT=2",
-                    text: "Task 2"
-                }, "occurence args is correct");
+            appointmentTooltipTemplate: function(data, index, targetedAppointmentData) {
+                assert.deepEqual(data, targetedAppointmentData);
             }
         });
 
-        $(this.instance.$element().find(".dx-scheduler-appointment").eq(1)).trigger("dxclick");
+        $(this.instance.$element().find(".dx-scheduler-appointment").eq(0)).trigger("dxclick");
         this.clock.tick(300);
     });
 
-    QUnit.test("dropDownAppointmentTemplate rendering args should be correct if displayedAppointmentDataField is specified", function(assert) {
+    QUnit.skip("the targetedAppointmentData parameter appends to arguments of the drop down appointment template for a recurrence rule", function(assert) {
         this.instance.option({
             dataSource: [{
-                startDate: new Date(2015, 4, 24, 9, 10),
-                endDate: new Date(2015, 4, 24, 11, 1),
-                recurrenceRule: "FREQ=DAILY;COUNT=2",
+                startDate: new Date(2015, 4, 24, 9),
+                endDate: new Date(2015, 4, 24, 11),
+                recurrenceRule: "FREQ=DAILY;COUNT=3",
                 allDay: true,
                 text: "Task 1"
             }, {
-                startDate: new Date(2015, 4, 24, 9, 10),
-                endDate: new Date(2015, 4, 24, 11, 1),
+                startDate: new Date(2015, 4, 24, 9),
+                endDate: new Date(2015, 4, 24, 11),
                 allDay: true,
-                recurrenceRule: "FREQ=DAILY;COUNT=2",
+                recurrenceRule: "FREQ=DAILY;COUNT=3",
                 text: "Task 2"
             }],
             maxAppointmentsPerCell: 1,
             currentDate: new Date(2015, 4, 24),
-            displayedAppointmentDataField: "Field",
             views: ["month"],
-            dropDownAppointmentTemplate: function(data) {
-                assert.deepEqual(data.Field.startDate, new Date(2015, 4, 25, 9, 10), "Start date of part is ok");
-                assert.deepEqual(data.Field.endDate, new Date(2015, 4, 25, 11, 1), "End date of part is ok");
+            dropDownAppointmentTemplate: function(itemData, itemIndex, itemElement, targetedAppointmentData) {
+                delete targetedAppointmentData.settings;
```

Comment:
What is checked regarding the 'delete' operation?

Reply:
I want to test fields from the expected object without settings object. So I am deleted this.
The settings have a lot of fields which it's not needed for the test.